commonutils.py

```diff
@@ -169,15 +169,15 @@ def setup_nrnbbcore_register_mapping(rings):
 
 def write_report_config(output_file, report_name, target_name, report_type, report_variable,
                         unit, report_format, target_type, dt, start_time, end_time, gids,
-                        buffer_size=8):
+                        buffer_size=8, scaling="Area"):
```

Comment:
Q: what are the possible/allowed values of the `scaling` argument (and possibly stupid Q, what is it referring to)? Referencing either the docs or using `typing.Literal` would be of great help.

Reply:
the scaling is mentioned here:

https://sonata-extension.readthedocs.io/en/latest/sonata_simulation.html#reports

atm it can be none or area.

I am not familiar with typing.Literal. May you be more specific of what you want to do? Create an python enum? Where should I put it?

Reply:
done